be/src/runtime/fragment_mgr.cpp

```diff
@@ -88,7 +89,7 @@ class RuntimeProfile;
 class FragmentExecState {
 public:
     FragmentExecState(const TUniqueId& query_id, const TUniqueId& instance_id, int backend_num, ExecEnv* exec_env,
-                      const TNetworkAddress& coord_hostport);
+                      const TNetworkAddress& coord_hostport, const std::string stream_load_label = "");
```

Comment:
`const std::string&` or `std::string`, otherwise the std::string will copy twice ?